.github/workflows/pr.yml

```
@@ -18,6 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       app-changes: ${{ steps.filter.outputs.app }}
+      docs-changes: ${{ steps.filter.outpus.docs }}
```

actionlint (1.7.4)

property "outpus" is not defined in object type {conclusion: string; outcome: string; outputs: object}

(expression)